tests/test_configuration_set.py

```diff
@@ -161,6 +199,21 @@ def test_configurations_load_xyz():
         assert config.mult == 2
 
 
+def test_configurations_load_numpy_compatibility():
+    """Test compatibility of mlp-train and npz files created with old version of numpy/autodE."""
+
+    file_path = os.path.join(here, 'data', 'water_01.npz')
+    data = ConfigurationSet(file_path)
+
```

Comment:
It would be good to assert a bit more about the loaded data. We can assert that there are two configurations, and that they have different sizes

```suggestion
    assert len(data) == 2
    assert data[0].n_atoms != data[1].n_atoms
```